docs/src/reference/asciidoc/core/intro/download.adoc

```diff
@@ -81,28 +69,14 @@ The Spark connector framework is the most sensitive to version incompatibilities
 | Spark Version | Scala Version | ES-Hadoop Artifact ID
 | 1.0 - 1.2 | 2.10 | <unsupported>
 | 1.0 - 1.2 | 2.11 | <unsupported>
-| 1.3 - 1.6 | 2.10 | elasticsearch-spark-13_2.10
-| 1.3 - 1.6 | 2.11 | elasticsearch-spark-13_2.11
-| 2.0+ | 2.10 | elasticsearch-spark-20_2.10
+| 1.3 - 1.6 | 2.10 | <unsupported> 
+| 1.3 - 1.6 | 2.11 |  <unsupported>
```

Comment:
Lets just combine these with the rows above to denote the combined range as unsupported.